### PR TITLE
Document connectivity metrics

### PR DESCRIPTION
**Pull request by maayanf24**
Related issue: https://github.com/submariner-io/submariner/issues/678

Signed-off-by: Maayan Friedman <maafried@redhat.com>

### CODE REVIEW

File: src/content/operations/monitoring/_index.en.md [hunk 1]
```diff
@@ -20,6 +20,24 @@ The following metrics are exposed currently:
...
   * `status`: the connection status (“connecting”, “connected”, or “error”)
+  
+Submariner exposed metrics - the following connectivity metrics shared the same set of labels:
+
+* `gateway_rx_bytes`: count of bytes received (by cable driver and cable)
+* `gateway_tx_bytes`: count of bytes transmitted (by cable driver and cable)
+* `connection_established_timestamp`: timestamp of last successful connection established (by cable driver and cable)
+* `connection_latency_seconds`: connection latency in seconds (average RTT, by cable driver and cable)
+
+  * `cable_driver`: the currentcable driver
```
Comment (dfarrell07) [inline src/content/operations/monitoring/_index.en.md:31]:
```suggestion
  * `cable_driver`: the current cable driver
```

File: src/content/operations/monitoring/_index.en.md [hunk 2]
```diff
@@ -20,6 +20,24 @@ The following metrics are exposed currently:
...
+  * `local_cluster`: the local cluster name
+  * `local_hostname`: the local hostname
+  * `local_endpoint_ip`: the local endpoint ip
```
Comment (dfarrell07) [inline src/content/operations/monitoring/_index.en.md:34]:
```suggestion
  * `local_endpoint_ip`: the local endpoint IP
```

File: src/content/operations/monitoring/_index.en.md [hunk 3]
```diff
@@ -20,6 +20,24 @@ The following metrics are exposed currently:
...
+  * `remote_cluster`: the remote cluster name
+  * `remote_hostname`: the remote hostname
+  * `remote_endpoint_ip`: the remote endpoint ip
```
Comment (dfarrell07) [inline src/content/operations/monitoring/_index.en.md:37]:
```suggestion
  * `remote_endpoint_ip`: the remote endpoint IP
```

File: src/content/operations/monitoring/_index.en.md [hunk 4]
```diff
@@ -20,6 +20,24 @@ The following metrics are exposed currently:
   * `remote_cluster`: the remote cluster name
   * `remote_hostname`: the remote hostname
   * `status`: the connection status (“connecting”, “connected”, or “error”)
+  
+Submariner exposed metrics - the following connectivity metrics shared the same set of labels:
```
Comment (dfarrell07) [inline src/content/operations/monitoring/_index.en.md:24]: Sorry for maybe being slow, but this sentence isn't clear to me. What's different between these metrics and the ones above? What does "shared by the same set of labels" mean?

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
Thanks! A few questions/nits inline.

File: src/content/operations/monitoring/_index.en.md (hunk 4 above)
Comment (maayanf24) [reply src/content/operations/monitoring/_index.en.md:24]: @dfarrell07 you're right. I need to find a better way to describe this.
First two metrics exposed by submariner-operator.
There are 5 new metrics - exposed by submariner
4 of them has the same set of labels.
The fifth metric has one more label on top of the previous labels.

Comment (dfarrell07) [reply src/content/operations/monitoring/_index.en.md:24]: Humm, yeah that is difficult to express clearly. Maybe some new subsections would help, like an H2 for "Exposed Metrics" and then H3s for "submariner" and "submariner-operator"?

I'm not familiar with the labels, but maybe extracting the "(by cable driver and cable)" bit into the sentence and listing the labels in parentheses would make sense?

**Review by dfarrell07 — APPROVED** [review]
LGTM, thanks @maayanf24!

File: src/content/operations/monitoring/_index.en.md [hunk 5]
```diff
@@ -9,6 +9,10 @@ weight = 20
 Submariner provides a number of Prometheus metrics, and sets up `ServiceMonitor` instances which allow these metrics to be scraped by an
 in-cluster Prometheus instance.
 
+## Exposed Metrics
+
+### Submariner-operator
```
Comment (nyechiel) [inline src/content/operations/monitoring/_index.en.md:14]:
```suggestion
### Submariner Operator
```

File: src/content/operations/monitoring/_index.en.md [hunk 6]
```diff
@@ -9,6 +9,10 @@ weight = 20
...
+
 The following metrics are exposed currently:
 
 * `submariner_gateways`: the number of gateways in the cluster;
```
Comment (nyechiel) [inline src/content/operations/monitoring/_index.en.md:18]:
```suggestion
* `submariner_gateways`: the number of gateways in the cluster
```

File: src/content/operations/monitoring/_index.en.md [hunk 7]
```diff
@@ -20,6 +24,26 @@ The following metrics are exposed currently:
   * `remote_cluster`: the remote cluster name
   * `remote_hostname`: the remote hostname
   * `status`: the connection status (“connecting”, “connected”, or “error”)
+  
+### Submariner
```
Comment (nyechiel) [inline src/content/operations/monitoring/_index.en.md:28]: @maayanf24 @skitt Technically, we expose the metrics for `submariner operator` and `submariner` separately as shown here. But should the user care about this? Should we just provide a list of all the available metrics or group them somehow by functionality?

Comment (skitt) [reply src/content/operations/monitoring/_index.en.md:28]: Right, the user shouldn’t care, all the metrics appear in the same Prometheus UI.

Comment (maayanf24) [reply src/content/operations/monitoring/_index.en.md:28]: I'll add them all in one list for now, later when we have some more we can think of grouping related metrics

**Review by nyechiel — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**